### PR TITLE
Exhibition hero updates

### PR DESCRIPTION
**Pull request by davidpmccormick**
## Type
🚑  Health

- [x] Demoed to @Heesoomoon @Norvard 

## Value
Pixel polish.

## Screenshot
screen shot 2017-09-14 at 10 32 16

### All
- [x] PR labelled and assigned
- [x] Any introduced code complexity has been flagged

### UI component only
- [x] A11y tested: `npm run test:accessibility <URL>`
- [x] Browser tested: `npm run test:browsers <URL>`
- [x] Works without JS in the client
- [x] Relevant tracking/monitoring has been considered

### CODE REVIEW

File: client/scss/base/_icons.scss [hunk 1]
```diff
@@ -19,10 +19,14 @@
...
 }
 
-.icon--halfx {
+.icon--point5x {
   height: 0.5em;
 }
 
+.icon--1point5x {
+  height: 1.5em;
+}
```
Comment (jamesgorrie) [inline client/scss/base/_icons.scss]: Design consistency 💨

File: client/scss/utilities/_root-scope-classes.scss [hunk 2]
```diff
@@ -292,6 +292,14 @@
   border-radius: $border-radius-unit;
 }
 
+.rounded-top-left {
+  border-top-left-radius: $border-radius-unit;
+}
+
+.rounded-bottom-right {
+  border-bottom-right-radius: $border-radius-unit;
+}
```
Comment (jamesgorrie) [inline client/scss/utilities/_root-scope-classes.scss]: We could create a class of `.round-opposites`, but I am fully aware I started this over abstraction thing...

File: server/filters/format-date.js [hunk 3]
```diff
@@ -32,17 +32,33 @@ export function formatDateRangeWithMessage({start, end}: {start: Date, end: Date
   const isPast = momentEnd.isBefore(momentNow);
   const isLastWeek = momentNow.isBetween(momentWeekBeforeEnd, momentEnd);
 
-  if (isFuture) {
+  return {
+    isFuture,
+    isPast,
```
Comment (jamesgorrie) [inline server/filters/format-date.js]: Could we not infer one from the other?
What about the `isPresent` state 🚎 ?

File: server/views/pages/exhibition.njk [hunk 4]
```diff
@@ -35,89 +35,97 @@
...
-
-              <div class="bg-white inline-block {{ {m:4} | spacingClasses({'padding': ['right', 'left']}) }} {{ {s:4} | spacingClasses({'padding': ['top', 'bottom']}) }}">
-                <div class="flex flex--v-center font-elf-green {{ {s:'HNM4', m:'HNM5', l:'HNM4'} | fontClasses }}">
-                  {% icon 'other/ticket', null, ['icon--2x', 'icon--teal'] %}
-                  <strong class="caps {{ {s: 2} | spacingClasses({margin: ['left']}) }} {{ {s: 6} | spacingClasses({margin: ['right']}) }}">Free</strong>
-                  {% if exhibition.end %}
+              <div class="bg-white inline-block exhibition-hero__box {{ {m:3, l:4} | spacingClasses({'padding': ['right', 'left']}) }} {{ {s:4} | spacingClasses({'padding': ['top', 'bottom']}) }} rounded-top-left rounded-bottom-right">
+                <div class="flex flex--v-center flex--h-space-between {{ {s:'HNM4', m:'HNM5', l:'HNM4'} | fontClasses }}">
+                  <span class="flex flex--v-center">
+                    {% icon 'other/ticket', null, ['icon--2x'] %}
```
Comment (jamesgorrie) [inline server/views/pages/exhibition.njk]: Passing around these icon classes makes me feel funny, but I'm not sure what we might do about it.

File: server/views/pages/exhibition.njk [hunk 5]
```diff
@@ -35,89 +35,97 @@
...
         </div>
-
       </div>
     </div>
 
 
-    {# {% componentV2 'image', exhibition.featuredImage, {}, {lazyload: true} %} #}
     <div class="is-hidden-s">
-      {% component 'wobbly-edge', {background: 'white'} %}
+      {% component 'wobbly-edge', {background: 'white', intensity: 30} %}
```
Comment (jamesgorrie) [inline server/views/pages/exhibition.njk]: What's intensity?

Comment (davidpmccormick) [reply server/views/pages/exhibition.njk]: It controls the maximum height of the wobbly line. Or how wobbly the wobbly line is, if you like.

File: server/filters/format-date.js (hunk 3 above)
Comment (davidpmccormick) [reply server/filters/format-date.js]: Yeah, I was wondering about this – I might pin this to another PR.

File: client/scss/utilities/_root-scope-classes.scss (hunk 2 above)
Comment (davidpmccormick) [reply client/scss/utilities/_root-scope-classes.scss]: I dilly-dallied on this – ultimately I just knew that as soon as we created a `round-opposites` class, there would be a requirement to e.g. only round the top-right corner, so settled on the two classes.

**Pull request merged**